modules/markup/external/external.go

```diff
@@ -96,7 +98,14 @@ func (p *Renderer) Render(ctx *markup.RenderContext, input io.Reader, output io.
 		args = append(args, f.Name())
 	}
 
-	cmd := exec.Command(commands[0], args...)
+	// FIXME this should use process
```

Comment:
doesnt it?

Reply:
It didn't - I wrote the FIXME then thought better of it and that I should just implement it. I thought I'd removed it.

```suggestion
```